src/controllers/users.controllers.js

```diff
@@ -19,4 +25,35 @@ export default class Users {
       return res.status(500).json(e);
     }
   }
+
+  /**
+   * Login a user.
+   * getting user information
+   * @param {Object} req valiable
+   * @param {Object} res valiable
+   * @return {Object} res
+   */
+  static async login(req, res) {
```

Comment:
you need to add try and catch in controllers in other to avoid crashing in server.